### PR TITLE
Feature: filter summary buttons links descriptions [ch1634]

### PR DESCRIPTION
**Pull request by joesb**
This cherry-picks three commits from #120 without the dependencies on previous commits from #118, meaning that I think it now supersedes #120.

@SamMed1 could you pull this PR locally to confirm that it adds the functionality you intended? If it does, then maybe either Lewis or I could give it a proper review.

### CODE REVIEW

**Comment by SamMed1** [comment]
Great, thanks very much.

Behaves as expected after running a `drush cr`

**Comment by joesb** [comment]
Cracking, I'll do a fuller code and functionality review for it now.

**Comment by SamMed1** [comment]
great, cheers @joesb

File: web/themes/custom/bevan/package-lock.json [hunk 1]
```diff
@@ -99,8 +99,8 @@
       "integrity": "sha512-5hYdAkZlcG8tOLujVDTgCT+uPX0VnpAH28gWsLfzpXYm7wP6mp5Q/gYyR7YQ0cKVJcXJnl3j2kpBan13PtQf6w==",
       "dev": true,
       "requires": {
-        "delegates": "1.0.0",
-        "readable-stream": "2.3.6"
+        "delegates": "^1.0.0",
+        "readable-stream": "^2.0.6"
```
Comment (joesb) [inline web/themes/custom/bevan/package-lock.json]: There's quite a lot of changes in `package-lock.json` including some odd looking requirements of above-a-much-lower-version for a package. Are these all kosher?

Comment (SamMed1) [reply web/themes/custom/bevan/package-lock.json]: @joesb - I haven't touched that so I'm a little confused on this to be honest, I don't understand why the project install changed the `settings.php` and this `.json` file, when they're not supposed to be.

Comment (joesb) [reply web/themes/custom/bevan/package-lock.json]: Ok, no probs. I'll take a look. There's a some merge conflicts now including in `styles.css`, so maybe we can revert this file and rebuild the `styles.css` file to handle merge conflicts in the stylesheets.

**Comment by joesb** [comment]
@SamMed1 and @lewisnyman can I ask for a quick final review from both you on the changes in the last two commits, as I'm trying to resolve merge conflicts bringing in changes from #119 merged into master into this branch — both your work edited the same file (`web/themes/custom/bevan/templates/facets/facets-summary-item-list.html.twig`), so I tried to unify those changes. I _think_ I managed it, but if you could both check that I've retained the changes you each made that'd be great.

It's really only that one file that needs a last quick 👍/ 👎 

Thanks, gents!

File: web/themes/custom/bevan/sass/components/_facets-checkbox.scss [hunk 2]
```diff
@@ -49,7 +49,10 @@
     margin: 1rem 0 1.5rem;
 
     &-row {
+      display: flex;
       line-height: 1.8rem;
+      margin-bottom: 0.45rem;
+
       &__spaced td {
```
Comment (lewisnyman) [inline web/themes/custom/bevan/sass/components/_facets-checkbox.scss]: I guess we can remove this code now?

**Comment by SamMed1** [comment]
Cheers @joesb  - Just pulled this onto my local, works as expected! Cheers

File: web/themes/custom/bevan/sass/components/_facets-checkbox.scss (hunk 2 above)
Comment (lewisnyman) [reply web/themes/custom/bevan/sass/components/_facets-checkbox.scss]: I meant to highlight the td line below

Comment (joesb) [reply web/themes/custom/bevan/sass/components/_facets-checkbox.scss]: Sorry, what bit can be removed? This bit https://github.com/cabinetoffice/national-leadership-centre-drupal/pull/121/files#diff-1ae830a3c7c2ba3843940d9d54be7fd9L53-L55

```scss

      &__spaced td {
        margin-bottom: 0.75rem;
      }
```

?

Comment (joesb) [reply web/themes/custom/bevan/sass/components/_facets-checkbox.scss]: Fixed with a9f5e4e

**Pull request merged**